charts/cloudzero-agent/README.md

```diff
@@ -79,124 +130,51 @@ prometheus-node-exporter:
   enabled: true
 ```
 
-This will deploy the necessary resources for metric scraping.
-
-#### Passing values to subcharts
-
-Values can be be passed to the metrics exporter subcharts by adding values according to the values specification for [kube-state-metrics](https://github.com/prometheus-community/helm-charts/blob/main/charts/kube-state-metrics/values.yaml) and [prometheus-node-exporter](https://github.com/prometheus-community/helm-charts/blob/main/charts/prometheus-node-exporter/values.yaml) in the `kube-state-metrics` and `prometheus-node-exporter` sections respectively.
-
-A common addition may be to pull the container images from custom image registries/repositories:
+#### Passing Values to Subcharts
 
-```yaml
-kube-state-metrics:
-  enabled: true
-  image:
-    registry: my-custom-registry.io
-    repository: my-custom-kube-state-metrics/kube-state-metrics
-
-prometheus-node-exporter:
-  enabled: true
-  image:
-    registry: my-custom-registry.io
-    repository: my-custom-prometheus/node-exporter
-```
+Values can be passed to subcharts like `kube-state-metrics` and `prometheus-node-exporter` by adding entries in `values-override.yaml` as per their specifications.
 
 ### Custom Scrape Configs
-If the chart is running **_without_** the `kube-state-metrics` and `prometheus-node-exporter` exporters enabled (meaning, those two exporters are deployed from some other source outside of this chart), then the scrape configs used by the underlying Prometheus agent may need to be adjusted.
 
-As an example, the out-of-the-box scrape config in this chart attempts to find the `kube-state-metrics` and `node-exporter` exporters via an annotation on k8s Services deployed by the KSM ande node-exporter subcharts. If those subcharts were instead deployed without any annotations, and were only available via Services with the addresses `my-kube-state-metrics-service.default.svc.cluster.local:8080` and `my-node-exporter.default.svc.cluster.local:9100`, we could add the following:
+If running without the default exporters, adjust Prometheus scrape configs:
 
-custom-scrape-config.yaml
+`values-override.yaml modifying subchart values`
 ```yaml
 prometheusConfig:
   scrapeJobs:
-    kubeStateMetrics: # this disables the default kube-state-metrics scrape job, which will be replaced by an entry in additionalScrapeJobs
+    kubeStateMetrics:
       enabled: false
     additionalScrapeJobs:
     - job_name: custom-kube-state-metrics
-      honor_labels: true
```

Comment:
I think streamlining these are great, and this is a great example to streamline. the downside is that I don't think this example scrape config would actually work in the way we would want it to. which is totally fine, the point of this section is just to show how to add scrape jobs. but I did add this specific scrape job for a user that wanted to figure out how to do this. 

doesn't have to be in this PR, but maybe we could move this sort of thing to an examples.md thing

Reply:
Will add it back

Reply:
thanks!

Reply:
it could totally go either way

Reply:
done!